source/fundamentals/builders/sort.txt

```diff
@@ -11,3 +11,347 @@ Sorts Builders
+:ref:`sort_example collection <sorts-builders-sort-example>`  
+in the following order: 
+
+.. code-block:: json
+
+   {"_id": 2, "letter": "a", "food": "donuts and coffee"}
+   {"_id": 3, "letter": "a", "food": "maple syrup"}
+   {"_id": 5, "letter": "a", "food": "milk and cookies"}
+   {"_id": 4, "letter": "b", "food": "coffee with sugar"}
+   {"_id": 1, "letter": "c", "food": "coffee with milk"}
+   {"_id": 6, "letter": "c", "food": "maple donut"}
+
+
+Descending
+~~~~~~~~~~
+
+A descending sort returns your documents from largest to smallest based
+on the value of a specified field. To specify a descending sort, use the
+``Sorts.descending()`` static factory method. Pass
+``Sorts.descending()`` the name of the field you need to sort on.
+
+The following code snippet shows how to specify a descending sort on the
+``_id`` field: 
+
+.. code-block:: java
+   
+   import static com.mongodb.client.model.Sorts.descending;
+
+   // <MongoCollection setup code here>
+   
+   collection.find().sort(descending("_id"));
+
+
+The code snippet above returns the documents in the
+:ref:`sort_example collection <sorts-builders-sort-example>`  
+in the following order: 
+
+.. code-block:: json
+
+   {"_id": 6, "letter": "c", "food": "maple donut"}
+   {"_id": 5, "letter": "a", "food": "milk and cookies"}
+   {"_id": 4, "letter": "b", "food": "coffee with sugar"}
+   ...
+
+Note that MongoDB does not guarantee sort order for documents that have
+fields with identical values. This means that if you pass "letter" to
+the ``descending()`` method, the first document returned could be any of
+the following documents:
+
+.. code-block:: json
+
+   {"_id": 1, "letter": "c", "food": "coffee with milk"}
+   {"_id": 6, "letter": "c", "food": "maple donut"}
+
+If you need to have a guaranteed sort order for documents that
+have fields with identical values you can specify additional fields to sort
+on in the event of a tie. 
+
+The following code snippet shows how to specify a descending
+sort on the ``letter`` field followed by the ``_id`` field: 
+
+.. code-block:: java
+
+   import static com.mongodb.client.model.Sorts.descending;
+
+   // <MongoCollection setup code here>
+   
+   collection.find().sort(descending("letter", "_id"));
+
+
+The code snippet above returns the documents in the
+:ref:`sort_example collection <sorts-builders-sort-example>`  
+in the following order: 
+
+.. code-block:: json
+
+   {"_id": 6, "letter": "c", "food": "maple donut"}
+   {"_id": 1, "letter": "c", "food": "coffee with milk"}
+   {"_id": 4, "letter": "b", "food": "coffee with sugar"}
+   {"_id": 5, "letter": "a", "food": "milk and cookies"}
+   {"_id": 3, "letter": "a", "food": "maple syrup"}
+   {"_id": 2, "letter": "a", "food": "donuts and coffee"}
+
+Combining Sort Criteria
+-----------------------
+
+To combine sort criteria, use the ``Sorts.orderBy()`` static factory
+method. The ``orderBy()`` method applies additional sort criteria from
```

Comment:
I think "additional" seems to imply that you should use something else as well as this for your sort criteria which is not the case. You could omit it, or replace it with something like "multiple". Also, similar to a comment in the Ascending section, the method is a builder that generates the Bson -- it doesn't apply the sort. That's what `sort()` does.
 
```suggestion
method. The ``orderBy()`` method applies sort criteria from
```